Header header
int16 eyesX
int16 eyesY
int16 jawX
int16 jawY
int16 eyeSkwint
int16 jawZ
int16 expression
bool blink
bool supprise
bool rightNose
bool leftNose